### PR TITLE
Fix SiCKRAGETV/sickrage-issues/issues/3219

### PR DESCRIPTION
**Pull request by fernandog**
Changes all log messages that return false to WARNING. So user can know why PP failed

@OmgImAlexis is this ok for you?

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

**Comment by miigotu** [comment]
@fernandog they should all be info not warning. The badging makes warning taboo.

**Comment by fernandog** [comment]
@OmgImAlexis ok to convert warning to info ? I don't use PP so to me does not matter

**Comment by miigotu** [comment]
Otherwise people who have extra files in their pp folder will get thousands of warnings logged and showing a badge overnight.

**Comment by fernandog** [comment]
Low space should we warnings

@OmgImAlexis need your feedback in all log messages. Which should be info or warning or debug.

**Comment by OmgImAlexis** [comment]
@miigotu I get that exact issue, I wake up to 300+ every day.

**Comment by fernandog** [comment]
So which should be warning in your opinion?

**Comment by WebSpider** [comment]
With the new warnings in UI, imo when something we do results in a problem for the user, it should be warning.

When we know we messed up, error

I have 2600 warnings in 2 days due to torrentproject.se being down ... I thought WWIII had started on my NAS, lol

**Comment by OmgImAlexis** [comment]
I think it should be a warning still but is there a way to just do a single error for the file instead of erroring every time the scan runs?

**Comment by fernandog** [comment]
I think we need a DB for that or create a empty file with same name but with .scanned extension and add this to ignore.

**Comment by OmgImAlexis** [comment]
@fernandog like what CP does?

**Comment by fernandog** [comment]
Dont know what they do. Is something similar? Db or new file?

**Comment by OmgImAlexis** [comment]
Example

````
media@media:/mnt/downloads/DONE/movie$ ls -la
total 10517337
drwxrwxrwx 1 media media       4096 Oct 13 11:55 .
drwxrwxrwx 1 media media      81920 Oct 13 15:33 ..
-rwxrwxrwx 1 media media  144589948 Sep 28 16:14 Burn-E (2008).mp4
-rwxrwxrwx 1 media media  262671942 Oct  9 02:47 Pikachu's Island Adventure (2006).mkv
-rwxrwxrwx 1 media media        192 Oct  9 02:55 Pikachu's Island Adventure (2006).unknown.ignore
-rwxrwxrwx 1 media media        198 Oct 13 11:55 Pixels.2015.1080p.BluRay.x264.YIFY.failed_rename.ignore
-rwxrwxrwx 1 media media 1766810714 Oct 13 11:52 Pixels.2015.1080p.BluRay.x264.YIFY.mp4
-rwxrwxrwx 1 media media        191 Oct 10 13:14 Pokemon-Mewtwo Strikes Back (1999) 720p DVDRip x264 Aac[Eng Subs]-atik0786 Silver RG.exists.ignore
-rwxrwxrwx 1 media media  679144687 Oct 10 13:01 Pokemon-Mewtwo Strikes Back (1999) 720p DVDRip x264 Aac[Eng Subs]-atik0786 Silver RG.mkv
-rwxrwxrwx 1 media media        191 Oct  8 11:13 Pokémon The First Movie Mewtwo Strikes Back (1998).exists.ignore
-rwxrwxrwx 1 media media 1287852527 Oct  9 01:32 Pokémon The First Movie Mewtwo Strikes Back (1998).mkv
-rwxrwxrwx 1 media media  943838305 Oct  9 02:02 Pokémon the Movie Genesect and the Legend Awakened (2013).mkv
-rwxrwxrwx 1 media media        191 Oct  9 08:07 Pokemon.the.Movie.White.Victini.and.Zekrom.2011.720p.BluRay.x264-WiKi.exists.ignore
-rwxrwxrwx 1 media media 4025538430 Oct  9 08:03 Pokemon.the.Movie.White.Victini.and.Zekrom.2011.720p.BluRay.x264-WiKi.mkv
-rwxrwxrwx 1 media media 1543329730 Oct  2 23:37 Shes.Funny.That.Way.2014.1080p.BluRay.x264.YIFY.mp4
-rwxrwxrwx 1 media media        192 Oct  8 11:09 Shes.Funny.That.Way.2014.1080p.BluRay.x264.YIFY.unknown.ignore
-rwxrwxrwx 1 media media  115482364 Oct  9 02:02 The Uncut Story of Mewtwo's Origin (2001).mkv
-rwxrwxrwx 1 media media     402944 May 23 11:15 Thumbs.db
````

**Comment by WebSpider** [comment]
We could also sort the errors/warnings for uniqueness before presenting then?

**Comment by fernandog** [comment]
Nice! They add the error + .ignore to file. User can see it and handle/fix issue.

**Comment by miigotu** [comment]
IMO adding files in the download location can get messy and can be avoided.

If you make them info and gradually move SOMe back to warning until its just right I think we will find an OK balance.

Ofc you can do like @WebSpider says and check if it is in the warning viewer first.

**Comment by fernandog** [comment]
check if it is in the warning viewer first would be nice but omly if we dont have thread name +counter

And we cant remove from log. Just not add a new warning indicator. 
Will fix other all warnins but need to check last 24/48 hours max. Otherwise a warning from two weeks ago will prevent a new warning to show up

**Comment by fernandog** [comment]
but I still think adding just a .ignore file would be better for PP issue

**Comment by miigotu** [comment]
just make them all info or debug. We can move individual ones back when we get questions.

You gotta stop trying to batch change log levels. Just change the ones that are causing spam.

File: sickbeard/postProcessor.py [hunk 1]
```diff
@@ -871,12 +871,11 @@ def process(self):
         # try to find the file info
         (show, season, episodes, quality, version) = self._find_info()
         if not show:
-            self._log(u"This show isn't in your list, you need to add it to SB before post-processing an episode",
+            self._log(u"This show isn't in your list, you need to add it to SR before post-processing an episode",
                       logger.WARNING)
```
Comment (miigotu) [inline sickbeard/postProcessor.py]: Make this debug  or info too..

Comment (fernandog) [inline sickbeard/postProcessor.py]: missed that

**Pull request merged**